doc/release/platforms/README.cygwin.rst

```diff
@@ -34,14 +34,13 @@ please let us know at chapel_info@cray.com.
 Required Packages
```

Comment:
I also thought it was wrong when I was first reviewing this file, but I tried it under cygwin to make sure (I did a fresh install of cygwin using the tarball to make sure the content in this file was still accurate.)